EXTENSIONS.md

```diff
@@ -0,0 +1,113 @@
+# Linkerd Extensions
+
+Linkerd has an extension model which allows 3rd parties to add functionality.
+Each extension consists of a binary CLI named `linkerd-name` (where `name` is
+the name of the extension) as well as a set of Kubernetes resources.  To invoke
+an extension, users can run `linkerd name` which will search the current PATH
+for an executable named `linkerd-name` and execute it.
+
+## Installing Extensions
+
+To install an extension, first download the extension executable and put it
+in your PATH. The extension can then be installed into your Kubernetes cluster
+by running `linkerd <extension name> install | kubectl apply -f -`. Similarly,
+it can be uninstalled by running
+`linkerd <extension name> uninstall | kubectl delete -f -`.
+
+A full list of installed extensions can be printed by running `linkerd check`.
```

Comment:
I will re-word this to clarify: this lists extensions that have been installed in the cluster.  I.e. it lists all namespaces with the `linkerd.io/extension` annotation.

We don't provide a mechanism for listing all extension executables on your path.  I'm not sure this is actually useful and also it will erroneously list things like `linkerd-stable-1.2.3` as extensions since they fit the naming scheme.

Reply:
I know this got updated on the website PR with listing by the label, but looks like it still needs to here as well.